tests/codeception/functional/Paragraphs/WYSIWYGCest.php

```diff
@@ -114,8 +114,20 @@ public function testEmbeddedVideo(FunctionalTester $I) {
     $I->click('Video', '.media-library-menu-video');
     $I->waitForElementVisible('.media-library-add-form-oembed-url');
     $I->fillField('Add Video via URL', 'https://www.youtube.com/watch?v=ktCgVopf7D0');
-    $I->click('Add');
+
+    // If the youtube api fails, lets try again after a few seconds.
+    $bail = 0;
+    while (!empty($I->grabMultiple('input.media-library-add-form-oembed-submit[value="Add"]'))) {
+      $I->click('Add');
+      $I->wait(5);
+      $bail++;
```

Comment:
LOL. Based on the conversation yesterday I get why we need this, but I kind of hate it.  :)